langchain/tsconfig.json

```diff
@@ -32,7 +32,8 @@
       "./prompt/index.ts",
       "./vectorstores/index.ts",
       "./text_splitter.ts",
-      "./index.ts"
+      "./index.ts",
+      "./memory/index.ts"
```

Comment:
There's a gotcha with this config here for typedoc, the root index.ts needs to always be the last file in the list, otherwise it misbehaves

Reply:
If only we could comment in `.json` 🙄 it's also annoying to duplicate this in the `package.json` and `tsconfig.json`, ideally the packages endpoint strategy would just pick up on our entrypoints automatically: https://typedoc.org/api/enums/EntryPointStrategy.html#Packages. Might be worth trying to upstream something

Reply:
roger